src/main/webapp/app/overview/course-lectures/course-lecture-details.component.ts

```diff
@@ -85,4 +90,12 @@ export class CourseLectureDetailsComponent implements OnInit, OnDestroy {
             this.isDownloadingLink = null;
         }
     }
+
+    onChildActivate(ref: StudentQuestionsComponent) {
+        this.studentQuestions = ref;
```

Comment:
this code is only invoked (see above) if this.studentQuestions is available. Why do we need to re-assign it here?

Reply:
see my other comment